proxy/juno/Dockerfile

```
@@ -0,0 +1,8 @@
+FROM docker.io/nethermind/juno:latest
+
+USER root
```

Hadolint

[warning] Last USER should not be root

(DL3002)